infrastructure/templates/template.json

```diff
@@ -532,7 +540,7 @@
         }
       ]
     },
-    "templateBaseUrl": "https://raw.githubusercontent.com/dfe-analytical-services/explore-education-statistics/master/infrastructure/templates/"
+    "templateBaseUrl": "[concat('https://raw.githubusercontent.com/dfe-analytical-services/explore-education-statistics/', parameters('branch'), '/infrastructure/templates/')]"
```

Comment:
yes I was thinking of doing similar :)

Reply:
cool cause this was the part I'm concerned about not working :)